app/services/migrations/remove_extra_derivatives.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Migrations
+  class RemoveExtraDerivatives
+    def self.call
+      new.run
+    end
+
+    def run
+      progress_bar
+      query_service.find_all_of_model(model: FileSet).each do |file_set|
```

Comment:
This isn't memory efficient - it'll load every file set into memory. See https://github.com/samvera-labs/valkyrie-sequel/blob/main/lib/valkyrie/sequel/query_service.rb#L43-L47.

We created the MemoryEfficientAll query for reindexing, but that doesn't let us filter to a specific model - it just has a deny list.